aicsimageio/tests/test_transforms.py

```diff
@@ -151,6 +353,88 @@ def test_reshape_data_values(data, given_dims, return_dims, idx_in, idx_out):
     assert type(actual) == type(slice_in)
 
 
+# Arrays used for value checking on kwarg provided reshape_data
+NP_ONES = np.ones((10, 10))
+TEST_NDARRAY = np.stack([NP_ONES * i for i in range(7)])
+DA_ONES = da.ones((10, 10))
+TEST_DARRAY = da.stack([DA_ONES * i for i in range(7)])
+
+
+@pytest.mark.parametrize(
+    "data, given_dims, return_dims, other_args, expected",
+    [
+        # Just dimension selection
+        (TEST_NDARRAY, "ZYX", "YX", {}, TEST_NDARRAY[0]),
```

Comment:
Same comment here, add these as an extra `parametrize` call so you reduce the number cases/duplicates you have to define